linter_config.json

```diff
@@ -19,7 +19,11 @@
   "Aggregate": true,
   "WarnUnmatchedNolint": true,
   "LineLength": 240,
-  "Exclude": ["comment or be unexported", "comment on exported"],
+  "Exclude": [
+    "redundant return statement",
```

Comment:
I think redundant return is to because that's a lint issue in one of the auto copy files. Instead of excluding this warning for all files can we just exclude the deepcopy file from lint checks?

Reply:
ok i will try to do a follow up PR

Reply:
Thanks. I  went ahead and submit this because it fixes the travis build